### PR TITLE
WIP: 2-cockpit-7: [no-test] images: Only run podman in debian.setup when it is available

### PR DESCRIPTION
**Pull request by mvollmer**
The ubuntu-2004 image doesn't have it.

 * [x] image-refresh ubuntu-2004
 * [x] image-refresh ubuntu-stable
 * [x] image-refresh ubuntu-2204
 - [ ] image-refresh debian-testing
 - [ ] image-refresh debian-stable

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-072424/

File: images/scripts/debian.setup [hunk 1]
```diff
@@ -248,5 +248,7 @@ ln -snf /bin/true /etc/network/if-up.d/openssh-server
 # sudo_root" for details.` message in admins terminal.
 touch /home/admin/.sudo_as_admin_successful
 
-# Pull podman containers
-/var/lib/testvm/podman-images.setup
+# Pull podman containers if we have podman installed
+if command -v podman >/dev/null; then
```
Comment (martinpitt) [inline images/scripts/debian.setup:252]: I'd have used a more static `if [ "$RELEASE" != "focal" ]` to explicitly rule out 20.04 -- we do expect this to work on all newer and future releases.

**Comment by mvollmer** [comment]
Image refreshes are only to prove that they still work. I don't think we need to install the resulting images.

**Review by martinpitt — COMMENTED** [review]
Alternate proposal which would restrict the refresh to ubuntu-2004 only. I don't like these dynamic conditions much, as they hide errors -- but if you prefer that, no veto.

File: images/scripts/debian.setup (hunk 1 above)
Comment (mvollmer) [reply images/scripts/debian.setup:252]: Yeah, I was thinking that the tests will make sure that the container images are actually there.  But if the tests are also "dynamic" and skip things when images are missing, then we are in trouble.

So I guess I agree that we should be more strict.

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-072424/

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-073131/

**Comment by cockpituous** [comment]
image-refresh ubuntu-2004 done: https://github.com/cockpit-project/bots/commits/image-refresh-ubuntu-2004-20220325-080909

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-080909/

**Comment by marusak** [comment]
So here was refresh and no tests were triggered. @martinpitt 
I think the ping on the other PR might be due to github having some troubles with sending the right events (github was troublesome yesterday afternoon).

**Comment by cockpituous** [comment]
image-refresh ubuntu-stable done: https://github.com/cockpit-project/bots/commits/image-refresh-ubuntu-stable-20220325-084444

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-9.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-114343/

**Comment by cockpituous** [comment]
image-refresh ubuntu-2204 done: https://github.com/cockpit-project/bots/commits/image-refresh-ubuntu-2204-20220325-125252

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-8.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-125353/

**Review by martinpitt — APPROVED** [review]

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-134545/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-134646/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-134646/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-135757/

**Comment by cockpituous** [comment]
image-refresh debian-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-testing-20220325-140101

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-8.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-140101/

**Comment by cockpituous** [comment]
image-refresh debian-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-testing-20220325-141717

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-3143-20220325-141717/

**Comment by cockpituous** [comment]
image-refresh debian-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-testing-20220325-145858

**Comment by cockpituous** [comment]
image-refresh debian-stable done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-stable-20220325-151010

**Pull request merged**